src/irmin-graphql/server.ml

```diff
@@ -350,6 +350,17 @@ struct
                 ~resolve:(fun _ (t, _) ->
                   let+ tree = Store.tree t in
                   Ok (tree, Store.Key.empty));
+              io_field "last_modified"
+                ~typ:(non_null (list (non_null (Lazy.force commit))))
+                ~args:
+                  Arg.
+                    [
+                      arg "key" ~typ:(non_null Input.key);
+                      arg "depth" ~typ:int;
+                      arg "number" ~typ:int;
```

Comment:
It looks like `arg "n"` is used elsewhere: https://github.com/mirage/irmin/blob/3a575b04255d4c96c651e712e849ee794fc2060f/src/irmin-graphql/server.ml#L665
I think it would be nice to remain consistent

Reply:
Right, good catch! Do you want me to bundle as a single commit, or apply these changes separately?

Reply:
I think we typically try to avoid commits for updating CHANGES and formatting, so a single commit would be ideal if it's not too much trouble.